src/screens/PinPad/PinPad.tsx

```diff
@@ -0,0 +1,113 @@
+import { useRef, useState, useEffect, useCallback } from 'react';
+import { SafeAreaView, StatusBar, StyleSheet, View } from "react-native";
+import ReactNativePinView from "react-native-pin-view";
+import BottomDrawer, {
+  BottomDrawerMethods,
+} from 'react-native-animated-bottom-drawer';
+import { Text, Icon } from "@components";
+import { faDeleteLeft, faUnlock } from "@fortawesome/free-solid-svg-icons";
+
+export const PinPad = (props) => {
+  const pinView = useRef(null);
+  const bottomDrawerRef = useRef<BottomDrawerMethods>(null);
+  const [showRemoveButton, setShowRemoveButton] = useState(false);
+  const [enteredPin, setEnteredPin] = useState("");
+  const [buttonPressed, setButtonPressed] = useState("");
+  const [showCompletedButton, setShowCompletedButton] = useState(false);
+  const [isVisible, setIsVisible] = useState(true);
+
+  const styles = StyleSheet.create({
+    container: {
+        backgroundColor:  "rgba(0,0,0,0.5)",
+    },
+    handleContainer: {
+      backgroundColor:  "transparent",
+    },
+    pinPad: {
+      backgroundColor: "transparent",
+      alignItems: "center",
+      flex: 1,
+    },
+    pinPadTitle: {
+      paddingTop: 48,
+      paddingBottom: 24,
+      color: "rgba(255,255,255,1)",
+      fontSize: 48,
+    },
+    whiteBorder: {
+      borderWidth: 1,
+      borderColor: "#FFF",
+    },
+  });
```

Comment:
We usually we `styled-components`.
Can you try to migrate this to `styled.ts`. Check other `styled.ts` files in the repo to see how it works. It's pretty simple!

And it help to keep the component's logic files clean.